projects/GEDKeeper2/GKUI/BaseWin.cs

```diff
@@ -1038,7 +1038,11 @@ public void ShowTips()
                         }
 
                         if (birthDays.Count > 0) {
-                            MainWin.Instance.Options.ShowTips = DayTipsDlg.ShowTipsEx(LangMan.LS(LSID.LSID_BirthDays), MainWin.Instance.Options.ShowTips, birthDays);
+                            MainWin.Instance.Options.ShowTips =
+                                DayTipsDlg.ShowTipsEx(

```

Comment:
I have one question: why this method's name is `ShowTipsEx`? Because the caller is `ShowTips`? No? 😃